docs/manual/common/guide/devmode/CassandraServer.md

```diff
@@ -44,6 +44,28 @@ In sbt:
 
 @[cassandra-clean-on-start](code/build-cassandra-opts.sbt)
 
+
+## Cassandra YAML configuration file
+
+The Cassandra server can be configured with an alternative YAML file. By default, Lagom development environment uses [dev-embedded-cassandra.yaml](https://github.com/lagom/lagom/blob/master/dev/cassandra-server/src/main/resources/dev-embedded-cassandra.yaml). This is a good default to quickly get started, but if you find yourself needing to start Cassandra with a different configuration, you can easily do so by adding your own Cassandra YAML file to you to your build.
+
```

Comment:
It's probably worth documenting the placeholders that get replaced when processing the config file above so that users know they can include them in their own config files:

```scala
       val amendedConf = conf
          .replace("$PORT", port.toString)
          .replace("$STORAGE_PORT", storagePort.toString)
          .replace("$DIR", cassandraDirectory.getAbsolutePath)
```

Reply:
good idead